### ISSUE TITLE
A11y_code.visualstudio.com_Docs Azure_Try Azure App Service_Keyboard:Keyboard focus is not visible on the "Microsoft Azure" link. 

### ISSUE THREAD

**Issue by Charanrajkb** [microsoft/vscode-docs#2802]
“[Check out Accessibility Insights! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](url)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 2.4.7

**Repro Steps:**
Navigate to the URL: [https://code.visualstudio.com/tryappservice/](url)
or
follow below steps:
1)Launch the application Url : https://code.visualstudio.com
2)Navigate to "Docs" link.
3)From left navigation, select "Azure'.
4)Navigate to "Try Azure App Service" link.
5)Using keyboard navigate with Tab key till the Microsoft Azure link.
6)Observe the keyboard focus.

**Actual:**
Keyboard focus is not visible on the "Microsoft Azure" link. 
**Issue is observed in Edge + Narrator and Chrome + NVDA.**

**Expected:**
Keyboard focus must be visible on the Microsoft Azure link.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
Low vision user will find difficulty in finding the keyboard focus on the Microsoft Azure link.

**MAS Reference:**
MAS2.4.7:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](url)

**Attachment Reference:**
[A11y_code.visualstudio.com_Docs Azure_Try Azure App Service_Keyboard.pptx](https://github.com/microsoft/vscode-docs/files/3320783/A11y_code.visualstudio.com_Docs.Azure_Try.Azure.App.Service_Keyboard.pptx)

**Comment by bowdenk7** [comment]
It appears to be a bug in Edge itself as this works fine in chrome and "new edge":
![image](https://user-images.githubusercontent.com/820883/64558010-bd86e480-d2f7-11e9-8c0d-1b7fda9abe00.png)
